cosmwasm/contracts/crosschain-swaps/src/contract.rs

```diff
@@ -60,7 +60,7 @@ pub fn execute(
             on_failed_delivery,
         } => {
             let swap_coin = cw_utils::one_coin(&info)?;
-            execute::swap_and_forward(
+            execute::unwrap_or_swap_and_forward(
```

Comment:
I guess this is a gas question, but do we think its better for all swaps to do the registry check?

I'm guessing the idea here is that the counterparty just "always" sends their token to swap to osmosis, as normal, with no thoughts wrt unwinding?